### PR TITLE
packmol (new formula)

### PR DESCRIPTION
**Pull request by agiliopadua**
### Have you:

- [y] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [y] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [y] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [y] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [y] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

File: packmol.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    bin.install("packmol")
+    pkgshare.install "solvate.tcl"
+    (pkgshare/"examples").install resource("examples")
+  end
+
+  test do
+    cp pkgshare/"examples/interface.inp", testpath
+    cp pkgshare/"examples/water.xyz", testpath
+    cp pkgshare/"examples/chlor.xyz", testpath
+    cp pkgshare/"examples/t3.xyz", testpath
```
Comment (dpo) [inline packmol.rb]: This could be
```
%w[interface.inp water.xyz chlor.xyz t3.xyz].each {|f| cp #{pkgshare}/"examples/#{f}", testpath}
```
(untested)

File: packmol.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class Packmol < Formula
+  desc "Packing optimization for molecular dynamics simulations"
+  homepage "http://www.ime.unicamp.br/~martinez/packmol/"
+  url "http://leandro.iqm.unicamp.br/packmol/versionhistory/packmol-16.261.tar.gz"
+  sha256 "6ef74005da672743eddfb54b7c231da1b2dd0d6757fdb3ac047110bac9d884cf"
+
+  env :std
```
Comment (dpo) [inline packmol.rb]: Why is this necessary?

File: packmol.rb (hunk 1 above)
Comment (agiliopadua) [inline packmol.rb]: Yes, with just a " in the right place.

File: packmol.rb (hunk 2 above)
Comment (agiliopadua) [inline packmol.rb]: the env:std is not necessary.

**Comment by dpo** [comment]
Thanks. Could you please squash your commits? Your commit message should say `packmol (new formula)`.

**Comment by agiliopadua** [comment]
I don't understand why there is a merge problem. It appeared after I rebased to squash commits.
